Unity3D/Assets/RosSharp/Scripts/ImagePublisher.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
```

Comment:
Please split the class ImagePublisher into the following 2 classes:
1 CameraImageManager:
* requires component typeof Camera
* calls CameraImagePublisher.Publish(Texture2D texture2D) every Update()

2 CameraImageManager
* requires component typeof RosConnector
* provides Public method Publish(Texture2D texture2D)